### PR TITLE
Double click changes the account

### PR DESCRIPTION
**Pull request by mohsinshabir1234**
Have added a  double click feature on accounts to change the accout.

### CODE REVIEW

File: gui/control.js [hunk 1]
```diff
@@ -43,13 +50,15 @@ function render() {
...
-            const error = await PSS.switchAccount(nickname);
-            if (!error) return;
-            showToast("Error!", errorToMessage[error])
+        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', function(){
+            switchaccounts(nickname);
         });
 
+        document.getElementById(`nick-${nickname}`).addEventListener('dblclick', function () {
+            switchaccounts(nickname);
+        });
```
Comment (adsau59) [inline gui/control.js:59]: use arrow function for clear code

File: gui/control.js [hunk 2]
```diff
@@ -28,6 +28,13 @@ async function addButtonPressed() {
...
     shell.openExternal(' https://definex.in/discord');
 }
+async function switchaccounts(nickname) {
+    logger.debug(`switching ${nickname}`)
+    const error = await PSS.switchAccount(nickname);
+    if (!error) return;
+    showToast("Error!", errorToMessage[error])
+    
+    }
 function render() {
```
Comment (adsau59) [inline gui/control.js:38]: correct indentation and add new line between functions for cleaner code

File: gui/style.css [hunk 3]
```diff
@@ -150,13 +150,19 @@ input{
...
 }
 .profile-img img{
   width: 100px;
   border: 2px solid white;
   border-radius: 50%;
   height: 100px;
 } 
+.profile-img:focus {
+  outline: none; /* Remove default outline on focus */
+  /* Add your custom focus styles here, for example: */
```
Comment (adsau59) [inline gui/style.css:163]: is this comment relevent to our code base?

**Review by adsau59 — CHANGES_REQUESTED** [review]

**Review by mohsinshabir1234 — COMMENTED** [review]
I have done the requested changes. Kindly check.

**Comment by mohsinshabir1234** [comment]
#66

File: gui/control.js [hunk 4]
```diff
@@ -18,61 +18,95 @@ window.addEventListener('unload', beforeQuit)
...
+const loadingOverlay = document.getElementById('loading-overlay');
+
+function showLoadingOverlay() {
+  loadingOverlay.style.display = 'flex';
+}
+
+function hideLoadingOverlay() {
+  loadingOverlay.style.display = 'none';
+}
+
```
Comment (adsau59) [inline gui/control.js:31]: create different pull request for loading issue

File: gui/control.js [hunk 5]
```diff
@@ -18,61 +18,95 @@ window.addEventListener('unload', beforeQuit)
...
+    document.getElementById(`delete-btn-${nickname}`).addEventListener('click', async function () {
+      const agreed = await showYesNoPopup(`Are you sure you want to delete ${nickname} account?`, "Cancel", "Delete Account");
+      if (!agreed) return;
+      logger.debug(`deleting ${nickname}`)
+      const error = await PSS.deleteAccount(nickname);
+      if (!error) return;
+      showToast("Error!", errorToMessage[error])
+    });
+  }
+}
```
Comment (adsau59) [inline gui/control.js:95]: please make sure changes doesn't include things u haven't changed

File: gui/control.js [hunk 6]
```diff
@@ -18,61 +18,95 @@ window.addEventListener('unload', beforeQuit)
...
 async function main() {
-
+  try {
+    showLoadingOverlay(); // Show loading overlay before the initialization process
```
Comment (adsau59) [inline gui/control.js:98]: why are we doing this in main?

**Review by adsau59 — CHANGES_REQUESTED** [review]

**Comment by adsau59** [comment]
duplicate of #68 closing

**Pull request closed without merging**